Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -1477,16 +1477,16 @@ void Renderer::FinishFrameData()
 void Renderer::RunFrameDumps()
 {
   Common::SetCurrentThreadName("FrameDumping");
-  bool dump_to_avi = !g_ActiveConfig.bDumpFramesAsImages;
+  bool dump_to_ffmpeg = !g_ActiveConfig.bDumpFramesAsImages;
   bool frame_dump_started = false;
 
-// If Dolphin was compiled without libav, we only support dumping to images.
+// If Dolphin was compiled without ffmpeg, we only support dumping to images.
 #if !defined(HAVE_FFMPEG)
-  if (dump_to_avi)
+  if (dump_to_ffmpeg)
   {
-    WARN_LOG(VIDEO, "AVI frame dump requested, but Dolphin was compiled without libav. "
-                    "Frame dump will be saved as images instead.");
-    dump_to_avi = false;
+    WARN_LOG(VIDEO, "FrameDump: Dolphin was not compiled with FFmpeg, using fallback option."
+                    "FrameDump: Frames will be saved as PNG images instead.");
```

Comment:
This will currently look like "FrameDump: Dolphin was not compiled with FFmpeg, using fallback option.FrameDump: Frames will be saved as PNG images instead." when printed. I'd suggest removing the second "FrameDump:" and having a space at the end of the first line.